core/crypto/src/signature.rs

```diff
@@ -160,7 +160,7 @@ impl PublicKey {
 
 // This `Hash` implementation is safe since it retains the property
 // `k1 == k2 ⇒ hash(k1) == hash(k2)`.
-#[allow(clippy::derive_hash_xor_eq)]
+#[allow(clippy::derived_hash_with_manual_eq)]
```

Comment:
This, I believe, should no longer be necessary, as the rename is a part of the changes that now allow manual implementation of hash regardless of how the `Eq` implementation is produced.

Reply:
Ahh, yes makes sense. Removed them.